Packs/USTA/Integrations/USTA/USTA.yml

```diff
@@ -0,0 +1,170 @@
+category: Data Enrichment & Threat Intelligence
+commonfields:
+  id: USTA
+  version: -1
+configuration:
+- defaultvalue: https://usta.prodaft.com
+  display: Server URL (e.g. https://usta.prodaft.com)
+  name: url
+  required: true
+  type: 0
+- additionalinfo: 'You can reach out your access token : https://usta.prodaft.com/#/api-documents'
+  display: API Key
+  name: apikey
+  required: true
+  type: 4
+- defaultvalue: "false"
+  display: Trust any certificate (not secure)
+  name: insecure
+  required: false
+  type: 8
+- defaultvalue: "false"
+  display: Use system proxy settings
+  name: proxy
+  required: false
+  type: 8
+description: 'USTA is an Cyber Intelligence Platform that responds directly and effectively
+  to today''s complex cyber threats. '
+display: USTA
+name: USTA
+script:
+  commands:
+  - arguments:
+    - auto: PREDEFINED
+      defaultValue: json
+      description: Format type of the returned result
+      name: format
+      predefined:
+      - json
+      - stix
+      - stix2
+      - txt
+    - description: Filtering by URL Address
+      name: url
+    - auto: PREDEFINED
+      default: true
+      defaultValue: "true"
+      description: You can search only those with or without domain name registration
+      name: is_domain
+      predefined:
+      - "true"
+      - "false"
+    - description: Filtering by malicious type
+      name: url_type
+    - description: 'Filtering by tags. Example: tag=Keitaro'
+      name: tag
+    - description: Starting date
```

Comment:
Please provide the time format here and give an example, true to all time arguments

Reply:
Resolved.